### ISSUE TITLE
anti-GPIHBP1 antibody

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10509]
**Preferred term label:**
anti-glycosylphosphatidylinositol anchored high-density lipoprotein binding protein 1 antibody positivity

**Synonyms**
anti-GPIHBP1 antibody positivity

**Definition (free text, please give PubMed ID)**
PMID: 28402248|36402671|35185060|32948662

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:444490

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT 
added as
HP:0025731
Anti-glycosylphosphatidylinositol anchored high-density lipoprotein binding protein 1 antibody positivity
